examples/bri-3/src/bri/identity/bpiSubjects/capabilities/updateBpiSubject/updateBpiSubject.command.ts

```diff
@@ -0,0 +1,8 @@
+export class UpdateBpiSubjectCommand {
```

Comment:
Question : why are we introducing public readonly variables? We agreed in the previous PR that variables will be private.